chrome/content/brief-overlay.js

```diff
@@ -217,11 +214,11 @@ var Brief = {
         })
 
         query.getEntryCount().then(unreadEntriesCount => {
-            Brief.statusCounter.value = unreadEntriesCount;
-            Brief.statusCounter.hidden = (unreadEntriesCount == 0);
-
-            // Attribute to enable custom styling via userChrome.css.
-            Brief.toolbarbutton.setAttribute('unread-entries', unreadEntriesCount);
+            let text = '';
+            if (unreadEntriesCount > 0)
+                // text length should be limited to 4 characters according to addon sdk documentation
+                text = Math.min(9999, unreadEntriesCount);
```

Comment:
It may be non-obvious to the users that `9999` really means "many". On the other hand, this way we don't have any problems with localization and translating "many" with the limit of 4 characters max. Maybe overflow it to `…678` to retain at least minimal utility?